example_camayoc.yaml

```diff
@@ -0,0 +1,130 @@
+---
+credentials:
+      # these credentials can be associated
+      # with sources by name in the inventory section
+    - name: root
+      type: 'network'
+      # NOTE: When running quipucords in a container, this path
+      # must be the path that exists in the container.
+      sshkeyfile: ~/.ssh/id_rsa
+      username: root
+    - name: 'vcenter'
+      type: 'vcenter'
+      password: 'example1'
+      username: 'username1'
+      rho: false
+    - name: 'sat6'
+      type: 'satellite'
+      password: 'CHANGEME'
+      username: 'CHANGEUSERNAME'
+      # When sharing a config file for quipucords and rho
+      # Mark credentials that you do not want to use in the rho
+      # tests with this flag
+      # All non-network sources must have this flag because
+      # rho does not know what to do with non-network credentials
+      rho: false
+
+qcs:
+    # this is how you would configure quipucords if running
+    # the development server
+    # if running in a container, set the port to 443
+    # and https to true. Keep ssl-verify false unless you've
+    # set up the certificates
+    hostname: '127.0.0.1'
+    https: false
+    port: 8000
+    ssl-verify: false
+    # credentials for logging into the server
+    username: 'admin'
+    password: 'pass'
+        # this section is being outmoded but is still
+        # currenlty used by some CLI tests
+        # if it is not present, some CLI tests will skip but nothing breaks
+        sources:
```

Comment:
I feel like we still want to be able to control the sources and here list which hosts (by hostname or even by a given name available on the inventory) should be included.

This will remove the possibility of specifying many hosts and credentials to be used. For example when we want to test if quipucords and/or rho will pick the second credential when the first one is invalid.

I mean having that control will benefit for creating more interesting test scenarios in the future.